scripts/download-realm.js

```diff
@@ -164,6 +167,7 @@ function getCoreRequirements(dependencies, options, required = {}) {
         case 'linux':
             required.CORE_SERVER_FOLDER += `/linux/${flavor}`;
             required.CORE_ARCHIVE = `realm-core-${flavor}-v${dependencies.REALM_CORE_VERSION}-Linux-devel.tar.gz`;
+            required.openssl = "https://static.realm.io/downloads/openssl/1.1.1b/Linux/x86_64/openssl.tgz";
```

Comment:
Maybe `openssl` -> `OPENSSL` to be consistent with the rest.

Reply:
👍